tests/population/test_data_transformations.py

```diff
@@ -48,7 +62,13 @@ def test_assign_demographic_proportions(pop_data, include_sex):
         )
 
 
-def test_rescale_binned_proportions_full_range(pop_data, include_sex):
+def test_single_group_assign_demographic_proportions(pop_data_single_group):
```

Comment:
I tried implementing this as a parameterization on the previous test but it got complicated w/ `include_sex` which is in turn a parameter. So I just made this separate/simple test